multimodal_knowledge_retriever/src/usage.ipynb

```diff
@@ -81,7 +81,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "qachain = multimodal.set_retrieval_chain(retriever)"
+    "qachain = multimodal.set_retrieval_chain(retriever, image_retrieval_type=\"summary\")"
```

Comment:
Is this notebook needed?  Should we move it from src to notebook?  Looks like possibly a remnant?

Reply:
this notebook shows the use of the multimodal class, unlike the other one which goes step by step, but you're right is misplaced , moved to notebooks folder